app/src/main/java/ch/epfl/skysync/screens/reports/ReportDetailsScreen.kt

```diff
@@ -0,0 +1,52 @@
+package ch.epfl.skysync.screens.reports
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.navigation.NavHostController
+import ch.epfl.skysync.components.CustomTopAppBar
+import ch.epfl.skysync.components.LoadingComponent
+import ch.epfl.skysync.viewmodel.FinishedFlightsViewModel
+
+@Composable
+fun ReportDetailsScreen(
+    flightId: String,
+    finishedFlightsViewModel: FinishedFlightsViewModel,
+    isAdmin: Boolean,
+    userId: String,
+    navController: NavHostController
+) {
+  val flight by finishedFlightsViewModel.getFlight(flightId).collectAsStateWithLifecycle()
+  val reportId by finishedFlightsViewModel.flightReports.collectAsStateWithLifecycle()
```

Comment:
Nitpick: should be `reportIds` as it is a list

Reply:
fixed